packages/solid/tsup.config.ts

```diff
@@ -1,31 +1,15 @@
-import { defineConfig } from 'tsup';
-// eslint-disable-next-line import/no-namespace
-import * as preset from 'tsup-preset-solid';
-
-const presetOptions: preset.PresetOptions = {
-  entries: [
-    {
-      entry: 'src/index.ts',
-    },
-  ],
-  // Set to `true` to remove all `console.*` calls and `debugger` statements in prod builds
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  drop_console: false,
-  // Set to `true` to generate a CommonJS build alongside ESM
-  // cjs: true,
-};
-
-export default defineConfig((config) => {
-  const watching = Boolean(config.watch);
-
-  const parsedOptions = preset.parsePresetOptions(presetOptions, watching);
-
-  if (!watching) {
-    const packageFields = preset.generatePackageExports(parsedOptions);
-
-    // will update ./package.json with the correct export fields
-    preset.writePackageJson(packageFields);
-  }
-
-  return preset.generateTsupOptions(parsedOptions);
-});
+import { solidPlugin } from 'esbuild-plugin-solid';
+import { defineConfig, type Options } from 'tsup';
+
+export default defineConfig((options: Options) => ({
+  entry: ['./src/index.ts'],
+  clean: true,
+  dts: true,
+  format: ['esm'],
+  tsconfig: './tsconfig.build.json',
+  external: ['solid-js'],
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  noExternal: Object.keys(require('./package.json').dependencies),
```

Comment:
why we're bundling the dependencies as well ?

Reply:
> why we're bundling the dependencies as well ?

react one is bundled with dependencies.
https://github.com/LottieFiles/dotlottie-web/blob/c9933892d8e90b60b138dc460741580356fe3a66/packages/react/tsup.config.cjs#L18

so i thought, we want it self-contained. no?

Reply:
hey @theashraf, I know you're busy, but is this gonna be silenced? lol, I'd prefer a response at least :)

Reply:
@moonlitgrace I'm so sorry, I missed your question :(

I don't think we should maintain a self-contained bundle for React either. We should update this approach. If I remember correctly, dotlottie-react was initially treated as an exception so we could easily use the bundle via CDNs in CodePen. but, I think this is no longer necessary.

Reply:
> dotlottie-react was initially treated as an exception so we could easily use the bundle via CDNs in CodePen.

oh now it make sense why its just react, not svelte or vue components.
resolved on https://github.com/LottieFiles/dotlottie-web/pull/373/commits/5f45f8c5cc1cfdfa9ed9095f2beeb432aebdef16, could you take a look?